webinterface/pages/DDA_Quant_ion.py

```diff
@@ -428,6 +426,15 @@ def create_submission_elements(self) -> None:
             st.session_state[self.variables_dda_quant.all_datapoints_submission] = st.session_state[
                 self.variables_dda_quant.all_datapoints
             ].copy()
+        if st.session_state[self.variables_dda_quant.input_df] is not None:
+            st.session_state[self.variables_dda_quant.input_df_submission] = st.session_state[
+                self.variables_dda_quant.input_df
+            ].copy()
+        if st.session_state[self.variables_dda_quant.result_perf] is not None:
+            st.session_state[self.variables_dda_quant.result_performance_submission] = st.session_state[
+                self.variables_dda_quant.result_perf
+            ].copy()
```

Comment:
Is it necessary to make a copy in memory?

Reply:
Yes, as the others are reset when you upload the params file.

Reply:
Probably it only create a new reference without copying the underlying data? We would need to check what the current default is in pandas